### PR TITLE
Update `eksctl delete` context help

### PR DESCRIPTION
**Pull request by onemorepereira**
### Description

Update the text that describes what `eksctl delete iamserviceaccount` does. It contained identical text to the `eksctl create` descriptor, which was incorrect and misleading. Please refer to issue #1529 for more details.

### Checklist

- [x] Added tests that cover your change (if possible)
- [x] Added/modified documentation as required (such as the `README.md`, and `examples` directory)
- [x] Manually tested

### CODE REVIEW

File: pkg/ctl/delete/iamserviceaccount.go [hunk 1]
```diff
@@ -23,7 +23,7 @@ func deleteIAMServiceAccountCmd(cmd *cmdutils.Cmd) {
 
 	var onlyMissing bool
 
-	cmd.SetDescription("iamserviceaccount", "Create an iamserviceaccount - AWS IAM role bound to a Kubernetes service account", "")
+	cmd.SetDescription("iamserviceaccount", "Delete an iamserviceaccount", "")
```
Comment (martina-if) [inline pkg/ctl/delete/iamserviceaccount.go]: Can you change `iamserviceaccount` for "IAM service account" which is more readable?

**Comment by martina-if** [comment]
@onemorepereira Thanks for your contribution! Sorry that we missed this for a few days.

**Comment by onemorepereira** [comment]
@martina-if glad I can contribute, even if just a bit. I'll get your suggestion implemented and update the PR!

File: pkg/ctl/delete/iamserviceaccount.go (hunk 1 above)
Comment (onemorepereira) [reply pkg/ctl/delete/iamserviceaccount.go]: @martina-if I just pushed and committed the suggested edit. Let me know if you spot anything else.

**Comment by martina-if** [comment]
@onemorepereira Can you please rebase this branch so I can merge it?

**Comment by onemorepereira** [comment]
> @onemorepereira Can you please rebase this branch so I can merge it?

@martina-if  rebase done and all pre-merge checks are.passing!

**Pull request merged**